src/components/MenuAuctions/index.tsx

```diff
@@ -72,8 +72,13 @@ export class MenuAuctions extends React.Component <MenuAuctionProps> {
                       key={`${auction.sell.address}-${auction.buy.address}-${i}`}
                     >
                       {auction.current && auction.current.dirRunning && auction.current.participatesNormal && (
-                        <tr onClick={() => push(`/auction/${auction.sell.symbol}-${auction.buy.symbol}-${auction.current.index}`)}>
-                          <td className="sectionLink">{auction.sell.symbol}/{auction.buy.symbol}</td>
+                        <tr>
+                          <td
+                            className="sectionLink"
+                            onClick={() => push(`/auction/${auction.sell.symbol}-${auction.buy.symbol}-${auction.current.index}`)}
```

Comment:
Why move onClick to `<td>`? Just because?

Reply:
When clicking claim it would trigger the location change

Reply:
makes sense :+1: